Remove unnecessary apport support files.

Ubuntu has an "Apport" problem-report system which Bazaar, being a Canonical
product, apparently has to support at the expense of being a usable system
anywhere else.

Author: Michael Homer <michael@gobolinux.org>

--- ./setup.py.how-broken-is-this	2010-08-07 11:35:57.000000000 +1200
+++ ./setup.py	2010-08-07 11:37:32.000000000 +1200
@@ -727,18 +727,8 @@
         # easy_install one
         DATA_FILES = [('man/man1', ['bzr.1'])]
 
-    if sys.platform != 'win32':
-        # see https://wiki.kubuntu.org/Apport/DeveloperHowTo
-        #
-        # checking the paths and hardcoding the check for root is a bit gross,
-        # but I don't see a cleaner way to find out the locations in a way
-        # that's going to align with the hardcoded paths in apport.
-        if os.geteuid() == 0:
-            DATA_FILES += [
-                ('/usr/share/apport/package-hooks',
-                    ['apport/source_bzr.py']),
-                ('/etc/apport/crashdb.conf.d/',
-                    ['apport/bzr-crashdb.conf']),]
+    # Apport is Ubuntu-specific garbage that has absolutely no place in
+    # generic software. Installing the files for it breaks the build.
 
     # std setup
     ARGS = {'scripts': ['bzr'],
